src/Scan/RestScans.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { CliInfo } from '../Config';
+import {
+  Discovery,
+  Module,
+  ScanConfig,
+  SourceType,
+  StorageFile
+} from './Scans';
+import { RestScans } from './RestScans';
+import { instance, mock, reset } from 'ts-mockito';
+import nock from 'nock';
+
+interface ResponseBody {
```

Comment:
Fir of all, you don't use it as a response body (this is the request body). Secondary, why don't re-use `ScanConfig` ?